vtexdocs/devportal
# feat(release-note): create release note page and component

#### What is the purpose of this pull request?

[More details ](https://www.notion.so/vtexhandbook/Criar-componente-release-notes-c179f61065b24471b9452284aef79c08)📖 

Create Release Notes page and components.

#### What problem is this solving?

We should have a page that shows recent releases. This page should be in timeline format, showing the latest releases with title, description and date. For this we created two components and the page. Please also follow the [additional documentation](https://www.notion.so/vtexhandbook/Criar-componente-release-notes-c179f61065b24471b9452284aef79c08), which contains some questions about the components of this update.

#### How should this be manually tested?

Please go to [this workspace,](https://deploy-preview-37--elated-hoover-5c29bf.netlify.app/updates/release-notes) and check that everything works correctly. Also compare it with the version in the [figma](https://www.figma.com/file/Lx6sXdrw9KEvtSEKWttmv8/Developer-Portal?node-id=129%3A28).
#### Screenshots or example usage
<img width="845" alt="image" src="https://user-images.githubusercontent.com/32322512/169291482-1d299c53-7b2a-46a9-8021-710dc5071d50.png">


#### Types of changes
